components/search/list-renderer/human-genomic-variant.js

```diff
@@ -0,0 +1,36 @@
+import {
+  SearchListItemContent,
+  SearchListItemMain,
+  SearchListItemMeta,
+  SearchListItemStatus,
+  SearchListItemTitle,
+  SearchListItemType,
+  SearchListItemUniqueId,
+} from "./search-list-item";
+import { PropTypes } from "prop-types";
+
+export default function HumanGenomicVariant({ item: variant }) {
+  return (
+    <SearchListItemContent>
+      <SearchListItemMain>
+        <SearchListItemUniqueId>
+          <SearchListItemType item={variant} />
+          {variant.uuid}
+        </SearchListItemUniqueId>
+        <SearchListItemTitle>
+          {variant.refseq_id ? variant.refseq_id : variant.reference_sequence}
```

Comment:
The equivalent could be `variant.refseq_id || variant.reference_sequence` — “if the first one is falsy, render the second”).